src/test/coverartutils_test.cpp

```diff
@@ -34,13 +34,15 @@ class CoverArtUtilTest : public MixxxTest, public CoverArtCache {
 
 const QString kCoverLocationTest("res/images/library/cover_default.png");
 const QString kTrackLocationTest(QDir::currentPath() %
-                                 "/src/test/id3-test-data/cover-test.mp3");
+                                 "/src/test/id3-test-data/cover-test-png.mp3");
 
 TEST_F(CoverArtUtilTest, extractEmbeddedCover) {
     const QString kTestPath(QDir::currentPath() % "/src/test/id3-test-data/");
     QImage cover;
     // We never need to acquire security tokens for tests since we don't run
     // them in a sandboxed environment.
+	
+    // TODO(XXX) we need to compare with a reference file to ensure that we are reading correct data
```

Comment:
1. humm, embedded cover is not the same in all files. mp3-png and opus are 64x64 with black corners. ogg and flac are 248x186 with black corners. The one you provided a link to is 64x64 with white corners (none of the test tracks have it embedded). I will need to make a whole set of test files with the same embedded reference image.
2. ok, I will just try, but don't expect this test to be  perfect on first try ;)
3. Ok, good.
4. Here we go...